scripts/contractInteraction/staking_vesting.py

```diff
@@ -415,4 +437,13 @@ def addVestingCodeHash(vestingLogic):
     data = staking.addContractCodeHash.encode_input(vestingLogic)
     sendWithMultisig(conf.contracts['multisig'], staking.address, data, conf.acct)
 
-
+# vestings is the array of Vesting struct
+# struct Vesting {
+#     uint256 vestingType;
+#     uint256 vestingCreationType;
+#     address vestingAddress;
+# }
+def registerVestingToVestingDetail(vestings):
```

Comment:
```suggestion
def registerVestingToVestingDetails(vestings):
    vestingRegistry = Contract.from_abi("VestingRegistryLogic", address=conf.contracts['VestingRegistryProxy'], abi=VestingRegistry.abi, owner=conf.acct)
    data = vestingRegistry.registerVestingToVestingDetails.encode_input(vestings)
    sendWithMultisig(conf.contracts['multisig'], vestingRegistry.address, data, conf.acct)
```

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/79f72c867c17cd1f12ed1e726ee79e2c308a8e9e)